internal/buffer/buffer.go

```diff
@@ -438,7 +447,6 @@ func (b *Buffer) Close() {
 			copy(OpenBuffers[i:], OpenBuffers[i+1:])
 			OpenBuffers[len(OpenBuffers)-1] = nil
 			OpenBuffers = OpenBuffers[:len(OpenBuffers)-1]
-			return
```

Comment:
Return was redundant in `CloseOpenBuffers()` but it is not exactly redundant here in `Close()`: it is to break from the loop once we've found our buffer and thus don't need to traverse `OpenBuffers` anymore.

Reply:
F...
I need to stop doing this while been distracted from the children. This is embarrassing :(

Reply:
No worries. :)